lib/PaymentService.js

```diff
@@ -0,0 +1,79 @@
+/**!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import axios from 'axios';
+
+/**
+ * Wrapper around the bedrock-payment-http API.
+ *
+ * @param {object} [options] - Options to use.
+ * @param {string} [options.baseURL = '/payment'] - The baseURL for
+ *   the payment api.
+ *
+ * @returns {PaymentService} The PaymentService.
+ */
+export default class PaymentService {
+  constructor({baseURL = '/payment'} = {}) {
+    this.baseURL = baseURL;
+  }
+
+  /**
+   * Gets payments for an account.
+   *
+   * @param {object} options - Options to use.
+   * @param {object} options.params - Query params for the request.
+   *
+   * @returns {Array<object>} All payments for an account.
+   */
+  async getPayments({params} = {}) {
+    const options = {params};
+    const {data} = await axios.get(this.baseURL, options);
+    return data;
+  }
+
+  /**
+   * Gets the meta-data necessary for a client side charge for a
+   *   payment gateway.
+   *
+   * @returns {Promise<object>} The meta data for the Payment Button.
+   */
+  async getMeta({service = 'paypal'} = {}) {
```

Comment:
This API says we're getting meta data but the created url looks like we're getting credentials. Is it the case that you're making an HTTP Request to receive secrets/credentials from the backend to complete charges on the client?

Reply:
I changed it to `getCredentials`. Sorry about this it was getMeta before renaming it. I'm assuming credentials is the right name for payment gateway oauth tokens, api keys, etc.

https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/ad2507115de81dcf96f940e9ce7172f59760240a

Reply:
@gannan08 I addressed all of your concerns if you are happy with the PR feel free to rebase it as @mattcollier has approved it.

Reply:
> I changed it to `getCredentials`. Sorry about this it was getMeta before renaming it. I'm assuming credentials is the right name for payment gateway oauth tokens, api keys, etc.
> 
> [ad25071](https://github.com/digitalbazaar/bedrock-web-payment/commit/ad2507115de81dcf96f940e9ce7172f59760240a)

Thank you for renaming it. What is in the response?

Reply:
for the accelerator `paypal-client-id`

Reply:
Are you saying this is the specific response for just the accelerator? Shouldn't this be the same response regardless of what product this gets built into? Additionally, we should not be using kebab case in our APIs.

Reply:
@gannan08 https://github.com/digitalbazaar/bedrock-payment-paypal/pull/2/commits/0eec0778ef91dcd8dcdb499321376e39d7f3ecc4